.github/workflows/reusable_changelog_reminder.yml

```diff
@@ -0,0 +1,42 @@
+# adapted from https://github.com/osmosis-labs/osmosis/blob/main/.goreleaser.yaml
+name: Changelog Reminder
+on:
+  workflow_call:
+    inputs:
+      run-changelog-check:
+        description: 'Run changelog check'
+        type: boolean
+        default: true
+
+jobs:
+  check_changelog:
+    name: Check Changelogs
+    runs-on: ubuntu-24.04
+    if: ${{ inputs.run-changelog-check }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v41.0.0
+        with:
+          files_ignore: |
```

Comment:
What do you think if `files_ignore` is considered as input?
As we use this workflow for many repos, there may be exceptions or additional files are needed to be ignored

Reply:
Good idea! applied